.github/workflows/ci.yml

```diff
@@ -49,4 +49,14 @@ jobs:
         run: |
           sudo systemctl restart apache2.service
           sudo cat /var/log/apache2/error.log
-
+      - name: Check error.log
+        run: |
+          # Send requests & check log format
+          # Valid request
+          curl -s http://127.0.01/ > /dev/null || echo $?
+          # Invalid request
+          curl -s http://127.0.01/%2e%2f > /dev/null || echo $? 
+          # Check log format
+          grep -F ModSecurity < /var/log/apache2/error.log | grep -vP "^\[[^\]]+\] \[security2:[a-z]+\] \[pid [0-9]+:tid [0-9]+\] (?:\[client [0-9.:]+\] )?ModSecurity" || exit 0
```

Comment:
I'm not sure this test cover the case what you wanted to fix, namely only one `[client]` field can be there - or you still accept the two fields log format?

This pattern:
```
\[pid [0-9]+:tid [0-9]+\] (?:\[client [0-9.:]+\] )
```
matches with this string:
```
[pid 6545:tid 6547] [client ::1:49050] [client ::1]
```
but only till the firs `[client]` field, so this substring:
```
[pid 6545:tid 6547] [client ::1:49050]
```

Was that your intention?

Reply:
`^\[[^\]]+\] \[security2:[a-z]+\] \[pid [0-9]+:tid [0-9]+\] (?:\[client [0-9.:]+\] )?ModSecurity` doesn't match `[pid 6545:tid 6547] [client ::1:49050] [client ::1] ModSecurity ...`

Reply:
Ah, you're right - that was my mistake.